projects/fhi-angular-components/src/styles/fhi-tree-views/_fhi-tree-view-checkbox.scss

```diff
@@ -1,21 +1,36 @@
 .fhi-tree-view-checkbox {
-  &__list {
-      list-style-type: none;
-  }
+    &__list {
+        list-style-type: none;
+    }
 
-  &__item {
-      position: relative;
-  }
+    &__item {
+        position: relative;
+    }
 
-  &__toggler {
-      position: absolute;
-      left: ($fhi-core-space-5) * -1;
-      background-color: $fhi-core-white;
-      border: 0;
-      padding: 0;
+    &__toggler {
+        position: absolute;
+        left: ($fhi-core-space-5) * -1;
+        background-color: $fhi-core-white;
+        border: 0;
+        padding: 0;
 
-      [class*="icon-"] {
-          background-size: $fhi-core-space-4;
-      }
-  }
+        [class*="icon-"] {
+            background-size: $fhi-core-space-4;
+        }
+    }
+
+    &__check-all {
+        border: 0;
+        color: $fhi-core-blue-1;
+        font-size: $fhi-core-font-size-2;
+        font-family: $fhi-core-font-family-sans-serif-regular;
+        height: unset;
+        margin: $fhi-core-space-1 0 $fhi-core-space-3;
+        padding: 0;
```

Comment:
For the keyboard users, I think I would prefer to have some padding ($fhi-core-space-1), and reduce margin the same amount (negative left margin)